app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -63,6 +83,10 @@ def redis_client
       @redis_client ||= IrsAttemptsApi::RedisClient.new
     end
 
+    def s3_helper
+      @s3_helper ||= JobHelpers::S3Helper.new
+    end
```

Comment:
since we're not using anything else from S3::Helper, maybe we could do:

```suggestion
    def s3_client
      @s3_client ||= JobHelpers::S3Helper.new.s3_client
    end
```

Reply:
Point taken!  Done.